### PR TITLE
Added a hyperlink to open a chat with our team

### PR DESCRIPTION
**Pull request by mkosma**

### CODE REVIEW

File: src/community/support.md [hunk 1]
```diff
@@ -21,4 +21,4 @@
   + [Sign up for Office Hours!](https://docs.google.com/forms/d/e/1FAIpQLSeMhZtzQQWB47I8HfPcJ5_pFyMhZO284PLIblDfshe30dEuXw/viewform)
 
 For fully managed services, we invite you to visit 
-[DataJoint](https://www.datajoint.com) for solutions and request more information.
+[DataJoint](https://www.datajoint.com) for solutions or to [chat with our teeam](./#hs-chat-open) for more information.
```
Comment (CBroz1) [inline src/community/support.md:24]: Hi @mkosma - Where are you trying to direct to? The `./#notation` directs to a heading within a page (example [webpage](https://datajoint.com/docs/core/concepts/mantra/) and [corresponding markdown](https://raw.githubusercontent.com/datajoint-company/datajoint-docs/main/src/core/concepts/mantra.md))

Comment (mkosma) [reply src/community/support.md:24]: Indeed. That nomenclature opens the chat widget on the current page.  See, e.g., https://community.hubspot.com/t5/Tips-Tricks-Best-Practices/How-to-Automatically-open-the-chat-widget-when-a-customer-loads/m-p/378386

Comment (kabilar) [inline src/community/support.md:24]:
```suggestion
[DataJoint](https://www.datajoint.com) for solutions or to [chat with our team](./#hs-chat-open) for more information.
```

**Comment by mkosma** [comment]
Kind of defeats the purpose of an easy "edit" button on the page. I don't have time for install/dev cycles, could one of you that has the environment built please test (or since it's simple, push live, test, iterate if needed)?

**Comment by CBroz1** [comment]
I checked the link and, while the chatbot doesn't launch in local deployment, it does change the URL to `http://localhost/docs/community/support/#hs-chat-open`

> @kabilar seems to kind of defeat the purpose of an easy "edit" button on the page. I don't have time for install/dev cycles, could one of you that has the environment built please test (or since it's simple, push live, test, iterate if needed)?

I think I would set a different standard for an outside or first-time contributor. I would check to support someone volunteering in that way, but would ask more of folks in-house

**Comment by mkosma** [comment]
Thanks. That looks right - and I see your point that it's not otherwise
testable in the local environment.

**Review by guzman-raphael — APPROVED** [review]
That is a neat feature! Nice find!

There is actually a really simple test you can do all without any dev setup. You can just manually modify a PROD url by adding the `#hs-chat-open` to the end of any link. Just tried it on my phone and it opened the chat. Not really necessary to test the 'linking' feature since that part is just standard markdown. Not sure why it didn't open for @CBroz1 locally. 🤔

> To the PR reviewer: please test that the link I added works properly.

@mkosma For the future, I don't feel this comment is necessary since it is part of the process anyway.

Reviewers should never blindly accept PRs since it affects the reviewer's reputation and this is well understood by others. That means there is natural validation either by the reviewer or through automated tests in our process. Your proposed changes here are based on what you've found and you are proposing that it works (based on the info you have). Although we recommend you validate it yourself to improve your own reputation, that doesn't remove the responsibility of the reviewer to validate. Expectation is that reviews should indicate if it passes (resulting in merge), if it needs work by requesting changes, or if additional info is needed.

**Comment by mkosma** [comment]
Thanks! Helpful to understand all that.

**Pull request merged**